### PR TITLE
use standard argument order (mu, sigma, tau) in NormalMixture

### PR DESCRIPTION
**Pull request by fredcallaway**
I tested it by running notebooks/marginalized_gaussian_mixture_model.ipynb

### CODE REVIEW

**Comment by fredcallaway** [comment]
Adresses #3400

**Comment by lucianopaz** [comment]
Hmm. The tests exceeded the maximum time available. We should address this in a separate PR. Once it's fixed, you can rebase this on top.

**Comment by twiecki** [comment]
Als needs a line in the release notes.

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 - `GLM.from_formula` and `LinearComponent.from_formula` can extract variables from the calling scope. Customizable via the new `eval_env` argument. Fixing #3382.
 
 ### Maintenance
-
+- Add `sigma`, `tau`, and `sd` to signature of `NormalMixture`.
```
Comment (twiecki) [inline RELEASE-NOTES.md]: Can you move that to the bottom instead?

File: pymc3/distributions/mixture.py [hunk 2]
```diff
@@ -564,12 +564,10 @@ class NormalMixture(Mixture):
...
-    def __init__(self, w, mu, comp_shape=(), *args, **kwargs):
-        if 'sd' in kwargs.keys():
-            kwargs['sigma'] = kwargs.pop('sd')
-
-        _, sigma = get_tau_sigma(tau=kwargs.pop('tau', None),
-                           sigma=kwargs.pop('sigma', None))
+    def __init__(self, w, mu, sigma=None, tau=None, sd=None, comp_shape=(), *args, **kwargs):
+        if sd is not None:
+            sigma = sd
+        tau, sigma = get_tau_sigma(tau=tau, sigma=sigma)
```
Comment (twiecki) [inline pymc3/distributions/mixture.py]:
```suggestion
        _, sigma = get_tau_sigma(tau=tau, sigma=sigma)
```

**Review by twiecki — APPROVED** [review]

**Review by lucianopaz — APPROVED** [review]

**Comment by lucianopaz** [comment]
Thanks a lot, @fredcallaway!

**Pull request merged**